inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -74,7 +76,7 @@ void LayerTestsCommon::Run() {
             } catch (const std::exception &ex) {
                 THROW_IE_EXCEPTION << "Incorrect target static shape: " << CommonTestUtils::vec2str(targetStaticShapes[i]) << std::endl << ex.what();
             }
-        }
+        } while (++i < targetStaticShapes.size());
```

Comment:
On line 77 - will it crash if exception occur if i==0 and targetStaticShapes is empty?

Reply:
Fixed